extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php

```diff
@@ -1,29 +1,7 @@
 <?php
 
 class ARecoveryEngineHooks {
-	/**
-	 * Register recovery related scripts on the top
-	 *
-	 * @param array $vars
-	 * @param array $scripts
-	 *
-	 * @return bool
-	 */
-	public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
-		global $wgDevelEnvironment;
-
-		if ( ARecoveryModule::isEnabled() ) {
-			$spStatusURL = 'http://community.wikia.com/api/v1/ARecoveryEngine/SourcePointStatus';
-			if ( !empty( $wgDevelEnvironment ) ) {
-				$spStatusURL = '/api/v1/ARecoveryEngine/SourcePointStatus';
-			}
-			$scripts .= '<script src="' . $spStatusURL . '"></script>';
-			$scripts .= F::app()->sendRequest( 'ARecoveryEngineApiController', 'getBootstrap' );
-		}
-
-		return true;
-	}
-
+	
 	public static function onBeforePageDisplay( &$outputPage, &$skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$outputPage, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)